example/next.config.js

```diff
@@ -2,4 +2,15 @@ module.exports = {
   future: {
     webpack5: true,
   },
+  async redirects() {
```

Comment:
Handling a global `/` redirect because when the user is authenticated, they need to navigate to another application's home page (still under the same domain though). This application would have a basePath set but keeping that out for simplicity.